vignettes/01_rphenoscape.Rmd

```diff
@@ -91,21 +91,21 @@ Note that because the KB integrates model organism phenotypes, the results show
 Now we will query for ‘Ictaluridae’ and ‘fin’ (including parts) to retrieve the list of studies that contain fin characters for this taxon.  
 
 ```{r}
-slist <- pk_get_study_list(taxon = "Ictaluridae", entity = "fin", relation = "part of")
+slist <- get_studies(taxon = "Ictaluridae", entity = "fin", includeRels = "part of")
 ```
 The list is returned as a data.frame containing the study IDs and citations. Navigate to Global Environment and click on ‘slist’ to view the ID and citation (author, year) for each study in the KB that contains any ictalurid species associated with fin data.  Note that taxon, entity, and quality are optional arguments in this query, and we have also included fin parts.  Here we found nine studies. Click on ‘slist’ in the Global Environment to view these studies and their IDs.
 
 Next we want to obtain the character matrix for one of these studies. Using the study IDs from the previous step, we can get a list of the character matrices as NeXML objects.  We will choose the Lundberg (1992) publication, the fifth item on the list above.   
 
 ```{r}
-lundberg_nexml <- pk_get_study_xml(slist$id[5])
+lundberg_nexml <- get_study_data(slist$id[5])
 ```
 Now go to Global Environment pane and click on ‘lundberg_nexml’ to view the study ID and associated NeXML object.  
 
 To view the full Lundberg (1992) matrix within RStudio, we need to first retrieve the matrix as a data.frame from the NeXML object:
 
 ```{r}
-lundberg_matrix <- pk_get_study(lundberg_nexml)
+lundberg_matrix <- lapply(lundberg_nexml, function(nex) get_char_matrix(nex))
```

Comment:
```suggestion
lundberg_matrix <- get_char_matrix(lundberg_nexml[[1]])
```

Reply:
This (indexing into the list instead of using `lapply()`) still seems to me the more straightforward way for teaching.